Sources/Interaction/Widgets/SphereHandleRepresentation/index.js

```diff
@@ -350,7 +350,11 @@ export function extend(publicAPI, model, initialValues = {}) {
   // Inheritance
   vtkHandleRepresentation.extend(publicAPI, model, initialValues);
 
-  macro.setGet(publicAPI, model, ['translationMode']);
+  macro.setGet(publicAPI, model, [
+    'translationMode',
+    'property',
```

Comment:
Is there a `property` in the model?

Reply:
Yes, line 338

Reply:
I don't see it...

Reply:
@jourdain , the property of SphereHandleRepresentation is defined here : https://github.com/finetjul/vtk-js/blob/e706894165f7afbd45fc57978b902226ba2c92ea/Sources/Interaction/Widgets/SphereHandleRepresentation/index.js#L338